_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,77 @@
+---
+name: image accessible name contains image file name
+description: |
+  This rule checks that the use of an image's file name in it's accessible name, is appropriate.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+The rule applies to any HTML `img` element that is exposed to assistive technologies, where the accessible name of the element contains the file name specified in the `src` attribute of the element.
+
+### Expectation
+Each test target has an accessible name that serves as an appropriate text alternative as described in [F30: Failure of Success Criterion 1.1.1 and 1.2.1 due to using text alternatives that are not alternatives (e.g., filenames or placeholder text)](https://www.w3.org/TR/WCAG20-TECHS/F30.html).
```

Comment:
I'm unsure if linking to outside tests is allowed in ACT RF. I suggest we make whatever part we want from F30 part of this expectation, rather than us referencing a document that can change at any time.